### PR TITLE
Presto connect plugin uri

### LINKED ISSUE #30: Make consul connect plugin source configurable
To support airgapped environments the artefact source uri for the plugin must be configurable

### PR DESCRIPTION
**Pull request by oschistad**
Fixes #30

### CODE REVIEW

**Comment by oschistad** [comment]
The test is failing because the presto service never goes online. In the coordinator logs I can see the following:
```
2020-10-02T05:44:06.940Z	ERROR	main	io.prestosql.server.Server	No service providers of type io.prestosql.spi.Plugin
java.lang.IllegalStateException: No service providers of type io.prestosql.spi.Plugin
	at com.google.common.base.Preconditions.checkState(Preconditions.java:589)
	at io.prestosql.server.PluginManager.loadPlugin(PluginManager.java:166)
	at io.prestosql.server.PluginManager.loadPlugin(PluginManager.java:157)
	at io.prestosql.server.PluginManager.loadPlugins(PluginManager.java:138)
	at io.prestosql.server.Server.doStart(Server.java:117)
	at io.prestosql.server.Server.lambda$start$0(Server.java:73)
	at io.prestosql.$gen.Presto_341____20201002_054313_1.run(Unknown Source)
	at io.prestosql.server.Server.start(Server.java:73)
	at io.prestosql.server.PrestoServer.main(PrestoServer.java:38)
```

This does not appear related to the changes I have made - could this be a new issue from upstream?

File: variables.tf [hunk 1]
```diff
@@ -87,12 +87,23 @@ variable "docker_image" {
...
+
 variable "consul_connect_plugin_version" {
   type = string
   description = "Version of the consul connect plugin for presto (on maven central) src here: https://github.com/gugalnikov/presto-consul-connect"
   default = "2.2.0"
 }
 
+variable "consul_connect_plugin_artifact_source" {
+  type = string
+  default = "https://oss.sonatype.org/service/local/repositories/releases/content/io/github/gugalnikov/presto-consul-connect/"
```
Comment (zhenik) [inline variables.tf:104]: host address is not correct
```suggestion
  default = "https://oss.sonatype.org/service/local/repositories/releases/content/io/github/gugalnikov/presto-consul-connect"
```

**Review by zhenik — CHANGES_REQUESTED** [review]

File: variables.tf [hunk 2]
```diff
@@ -82,12 +82,23 @@ variable "docker_image" {
...
+
 variable "consul_connect_plugin_version" {
   type = string
   description = "Version of the consul connect plugin for presto (on maven central) src here: https://github.com/gugalnikov/presto-consul-connect"
   default = "2.2.0"
 }
 
+variable "consul_connect_plugin_artifact_source" {
+  type = string
+  default = "https://oss.sonatype.org/service/local/repositories/releases/content/io/github/gugalnikov/presto-consul-connect"
```
Comment (zhenik) [inline variables.tf:99]:
```suggestion
  description = "Artifact URI source"
  default = "https://oss.sonatype.org/service/local/repositories/releases/content/io/github/gugalnikov/presto-consul-connect"
```

**Review by zhenik — CHANGES_REQUESTED** [review]

**Comment by zhenik** [comment]
blocker #35 

vagrant-hashistack 0.4.3 require intentions creation once consul_acl=enabled and default_policy=deny. It iwll be fix in #35 .

**Review by zhenik — APPROVED** [review]

**Pull request merged**